opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/TaskRegisterPresenter.java

```diff
@@ -67,4 +68,13 @@ public void onFormSaveFailure(String eventType) {
         view.hideProgressDialog();//register will refresh on resume
     }
 
+    @Override
+    public void onInterventionFormDetailsFetched(CardDetails finalCardDetails) {
+        // do nothing for now
+    }
+
+    @Override
+    public void onCardDetailsFetched(CardDetails cardDetails) {
```

Comment:
You should remove this method from org.smartregister.reveal.contract.BaseContract.BasePresenter
and define in org.smartregister.reveal.contract.ListTaskContract.Presenter

Reply:
Done.